cmd/generate.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	pkgcmd "github.com/surajssd/kapp/pkg/cmd"
+)
+
+var (
+	AppFiles []string
```

Comment:
I thought `GenerateFiles` as well first (and even used it in my now closed PR) but after reading @kadel 's PR, I think, the correct choice of name is `AppFiles`. Unless there is a convention that we should keep the variables similar or related to the command, I would vote for AppFiles because these files (app files) are the input to the `generateCmd`.